src/evict/evict_page.c

```diff
@@ -227,9 +234,15 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
      * We have loaded the new disk image and updated the tree structure. We can no longer fail after
      * this point.
      */
-    session->evict_timeline.evict_finish = __wt_clock(session);
-    eviction_time =
-      WT_CLOCKDIFF_US(session->evict_timeline.evict_finish, session->evict_timeline.evict_start);
+    if (session->evict_timeline.nested_eviction) {
```

Comment:
We probably should rename nested_eviction to reentry_hs_eviction.

Reply:
done here: [182a7db](https://github.com/wiredtiger/wiredtiger/pull/9055/commits/182a7db1c07ad39d9f55d68611382ff4baedcafc)